tests/components/DigitalTerrainWebContainer.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import DigitalTerrainWebContainer from '../../src/components/DigitalTerrainWebContainer';
+
+describe('DigitalTerrainWebContainer', () => {
+  let container;
+  beforeEach(() => {
+    container = shallow(<DigitalTerrainWebContainer />);
```

Comment:
is `shallow` just `render` but without child components? Or does it have additional/different behavior?
Edit: I see that your second test below doesn't fully render the container but expects it to have a child element. Guess I'll look up what `shallow` does now 🤔

Reply:
Ya, Render, Shallow, and Mount are all different things and of varying degrees how much of the React lifecycle methods it uses. In general I use:
Render: To see just the dom when I don't care what lifecycle methods React will be using
Shallow: I want to see some of the lifecycle methods in the beginning, like Render(), and the first layer or two of the Dom
Mount: I want to see everything and I may be doing something in this test that could cause an update and I want React to be aware so I can test for it.